test/rqrcode/export_html_test.rb

```diff
@@ -0,0 +1,11 @@
+require 'test_helper'
+
+describe 'Export::HTML' do
+  it 'must respond_to html' do
+    RQRCode::QRCode.new('html').must_respond_to :'as_html'
+  end
+
+  it "must export to html" do
+    RQRCode::QRCode.new('html').as_html.must_match(/<table>.+<\/table>/)
+  end
```

Comment:
Is it worth adding a more specific content match test, adding an AS_HTML entry in `test/rqrcode/data.rb`, so that we're testing the row / cell formatting as well as the general table format?

Reply:
Yes, you're right. I'd completely forgotten. I'd changed all the other specs.